commit.go

```diff
@@ -169,7 +170,7 @@ func (c *Commit) Decode(from io.Reader, size int64) (n int, err error) {
 	c.Message = strings.Join(messageParts, "\n")
 
 	if err = s.Err(); err != nil {
-		return n, err
+		return n, fmt.Errorf("failed to parse commit bufffer: %s", err)
```

Comment:
Minor nitpick: buffer only takes two 'f' :wink:

Reply:
Fixed.  I'm going to blame my MacBook Pro keyboard.

Reply:
The bigger the buffer, the more f's it has!

Reply:
Oh, the dreaded _butterfffly_ keyboard :laughing: